xmipp

```diff
@@ -1330,7 +1401,6 @@ if __name__ == '__main__':
         else:
             Nproc=8
         compileDependencies(Nproc)
-        downloadDeepLearningModels("build")
```

Comment:
is this on purpose?

Reply:
Yes, now it's downloaded with the deepLearningToolkit.